wolfssl/wolfcrypt/types.h

```diff
@@ -438,6 +438,8 @@
 
     #ifdef WOLFSSL_RIOT_OS
         #define EXIT_TEST(ret) exit(ret)
+    #elif defined(HAVE_STACK_SIZE)
+        #define EXIT_TEST(ret) return 0
```

Comment:
@toddouska : pushed fixes for these return codes. Did a rebase/squash merge and resolved conflict in benchmark.